optimizer-fix-controls.md

```diff
@@ -71,6 +71,13 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 - This variable sets the threshold for the optimizer's heuristic strategy to select access paths. If the estimated rows for an access path (such as `Index_A`) is much smaller than that of other access paths (default `1000` times), the optimizer skips the cost comparison and directly selects `Index_A`.
 - `0` means to disable this heuristic strategy.
 
+### [`47400`](https://github.com/pingcap/tidb/issues/47400) <span class="version-mark">New in v8.4.0</span>
+
+- Default value: `ON`
+- Possible values: `ON`, `OFF`
+- This variable sets the minimum row estimate that the optimizer will allow. Due to challenges that all optimizers face in accuracy in estimating the number of qualified rows for each plan step in a query plan, it is common that an optimizer will set a lower limit. The default (`ON`) implemented in TiDB is the new behavior and is consistent with other databases.
+- `OFF` means disable this minimum value, which is the prior behavior (before this enhancement and fix-control).
```

Comment:
```suggestion
- This variable controls whether to set a minimum row estimate (`estRows`) of 1 for the optimizer. The optimizer often sets a lower limit for `estRows` due to challenges in accurately estimating the number of qualified rows for each plan step in a query plan.
- `ON`: sets a minimum `estRows` of 1. This is the new behavior introduced in v8.4.0 and aligns with other databases.
- `OFF`: disables the minimum row estimate limit and maintains the behavior of versions before v8.4.0. In this case, `estRows` might be zero.
```
@terry1purcell I've refined the description based on the GitHub issue details for improved clarity and precision. Please review the changes.